ingestion_server/test/run_test.sh

```diff
@@ -15,7 +15,6 @@ else
 fi
 
 PYTHONWARNINGS="ignore:Unverified HTTPS request" \
-PYTHONPATH=. \
```

Comment:
Defining this isn't necessary as this file is run via `pipenv run` and pipenv already sets the correct python path for you. Leaving this in made the tests fail on my local.